src/main/scala/com/amazon/deequ/suggestions/rules/FractionalCategoricalRangeRule.scala

```diff
@@ -23,16 +23,17 @@ import com.amazon.deequ.metrics.DistributionValue
 import com.amazon.deequ.profiles.ColumnProfile
 import com.amazon.deequ.suggestions.ConstraintSuggestion
 import com.amazon.deequ.suggestions.ConstraintSuggestionWithValue
+import com.amazon.deequ.suggestions.rules.FractionalCategoricalRangeRule.defaultIntervalStrategy
+import com.amazon.deequ.suggestions.rules.interval.{ConfidenceIntervalStrategy, WilsonScoreIntervalStrategy}
```

Comment:
nit: Could we avoid grouped imports and use one import per line?

Reply:
Just to clarify, do we prefer separate import or single import but with each on a single line
```scala
import com.amazon.deequ.suggestions.rules.interval.ConfidenceIntervalStrategy
import com.amazon.deequ.suggestions.rules.interval.WilsonScoreIntervalStrategy
```
or
```scala
import com.amazon.deequ.suggestions.rules.interval{
  ConfidenceIntervalStrategy,
  WilsonScoreIntervalStrategy
}
```

Reply:
The former. It helps with automatic resolution of merge conflicts.

Reply:
Updated